src/documents/UserInterfaceOptionsAPI.md

```diff
@@ -163,6 +163,45 @@ var uio = fluid.uiOptions.prefsEditor(".myContainer", {
 
 The options supported by UI Options are described below.
 
+### defaultLocale
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>defaultLocale</code> option allows you to specify the default locale and language of the UI Options interface. The locale specified must have associated JSON message files present in the source tree of Infusion at the path <code>src/framework/preferences/messages/</code>. The following locales are currently available in the latest version of Infusion: English (CA), English (US), Farsi, French, Spanish.
```

Comment:
I guess the path for the message bundles isn't restricted to that location. It should be configurable through the terms in the options.